core/src/main/java/bisq/core/provider/mempool/TxValidator.java

```diff
@@ -85,13 +85,13 @@ public TxValidator(DaoStateService daoStateService,
                        String txId,
                        Coin amount,
                        @Nullable Boolean isFeeCurrencyBtc,
```

Comment:
Better remove the @Nullable and use primitive boolean value as the callers always provide a non null value.